testing/files/eth-genesis.json

```diff
@@ -1,358 +1,39 @@
 {
   "config": {
-    "chainId": 80087,
+    "chainId": 1234,
     "homesteadBlock": 0,
-    "daoForkBlock": 0,
-    "daoForkSupport": true,
     "eip150Block": 0,
     "eip155Block": 0,
     "eip158Block": 0,
     "byzantiumBlock": 0,
     "constantinopleBlock": 0,
     "petersburgBlock": 0,
     "istanbulBlock": 0,
-    "muirGlacierBlock": 0,
     "berlinBlock": 0,
     "londonBlock": 0,
-    "arrowGlacierBlock": 0,
-    "grayGlacierBlock": 0,
     "mergeNetsplitBlock": 0,
-    "shanghaiTime": 0,
-    "cancunTime": 0,
     "terminalTotalDifficulty": 0,
     "terminalTotalDifficultyPassed": true,
-    "ethash": {}
+    "shanghaiTime": 0,
+    "cancunTime": 0
   },
-  "coinbase": "0x0000000000000000000000000000000000000000",
-  "difficulty": "0x0",
-  "extraData": "0x0000000000000000000000000000000000000000000000000000000000000000658bdf435d810c91414ec09147daa6db624063790000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
-  "gasLimit": "0x1c9c380",
-  "nonce": "0x0000000000000000",
+  "nonce": "0x0",
   "timestamp": "0x0",
+  "extraData": "0x",
+  "gasLimit": "0x1c9c38000",
+  "difficulty": "0x0",
+  "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+  "coinbase": "0x0000000000000000000000000000000000000000",
   "alloc": {
-    "0xaF9053bB6c4346381C77C2FeD279B17ABAfCDf4d": {
```

Comment:
I think this address has the private key defined in the .env.example file which is useful. 
Would be nice to keep it for testing purposes maybe. 

I'm unsure how I would get the priv keys for the two addresses in the `alloc` block here unless we can add them somewhere in the repo or another env file.

Reply:
great catch, added it back!